src/sass/utilities/_spacing.scss

```diff
@@ -27,7 +27,9 @@ $spacing-sizes: (
   -md: $spacing-md,
   -lg: $spacing-lg,
   -xl: $spacing-xl,
-  -xxl: $spacing-xxl
+  -xxl: $spacing-xxl,
+  -3-x: $spacing-3-x,
```

Comment:
My expectation from how sizes are written in other contexts (like clothing) was that these classes would be `3xl`, `4xl`, etc.

Reply:
@scottanthonymurray Yeah, that's a good point. I agree. For some reason when I tried that to output was getting translated to `-3-x` and `-4-x` when it's converted from the main `tokens/spacing.json` file to the Sass variables so I opted to match that here in the Sass map that we use to generate all the utility classes. Maybe I'll take a quick look at how were processing the transformation and see if I can get it to work correctly.

Reply:
@scottanthonymurray It looks like due to limitations in the way style dictionary processes property names. The best we can do for the new spacing tokens is `-3-xl` and `4-xl`. Not exactly what I wanted but I think it's workable until we can circle back and think of a better way to handle it.